spec/features/address_bar_spec.rb

```diff
@@ -10,6 +10,9 @@ def query_string
   context 'when loading collection details page from search page' do
     before :all do
       visit '/search'
+      if page.has_link?('closeInitialTourModal')
```

Comment:
@mightynimble - this was a million times true - has_css is so much faster.  Thanks!